src/Models/OrderRequest.php

```diff
@@ -81,15 +92,31 @@ public function setOrderDelivery(OrderDelivery $OrderDelivery)
 
     public function toArray()
     {
-        return array(
+        return [
             'OrderHeader' => $this->OrderHeader->toArray(),
             'OrderLines' => $this->OrderLines,
             'OrderDelivery' => $this->OrderDelivery->toArray()
-        );
+        ];
     }
 
     public function toJson()
     {
         return json_encode($this->toArray());
     }
-}
\ No newline at end of file
+
+    /**
+     * Specify data which should be serialized to JSON
+     * @link http://php.net/manual/en/jsonserializable.jsonserialize.php
+     * @return mixed data which can be serialized by <b>json_encode</b>,
+     * which is a value of any type other than a resource.
+     * @since 5.4.0
+     */
+    function jsonSerialize()
+    {
+        return [
+            'OrderHeader' => $this->OrderHeader->toArray(),
```

Comment:
Removed and implemented jsonserializable in commit e1b3a962c4ee8813a9e17fefe76da850bffa0b05